doc/man7/EVP_KDF-X942-ASN1.pod

```diff
@@ -77,6 +77,28 @@ Valid values are "AES-128-WRAP", "AES-192-WRAP", "AES-256-WRAP" and "DES3-WRAP".
 
 =back
 
+The OpenSSL FIPS provider also supports the following parameters:
+
+=over 4
+
+=item "fips-indicator" (B<OSSL_KDF_PARAM_FIPS_APPROVED_INDICATOR>) <integer>
+
+A getter that returns 1 if the operation is FIPS approved, or 0 otherwise.
+This may be used after calling EVP_KDF_derive. It returns 0 if any "***-check"
```

Comment:
Unless this is going to have others just use key-check here.

Reply:
I believe this will still need output key length check as raised in https://github.com/openssl/openssl/pull/25507 but I can change it to just key-check for now.

(full disclosure I copy & pasted this documentation snippet ;-) )